src/game_api/entity_db.hpp

```diff
@@ -86,7 +86,7 @@ struct EntityDB
     int32_t tile_y;
     uint8_t damage;
     uint8_t life;
-    uint8_t field_96;
+    uint8_t sacrifice_value;
```

Comment:
Comment? is that a amount of favor you get for sacrifice? (last time i checked that was hardcoded)